### ISSUE TITLE
Padding modes

### ISSUE THREAD

**Issue by xir4n** [kymatio/murenn#1]
Problem related to the DTCWTForward output length
Description of the problem:
When decomposing tensors using DTCWTForward, the length of the input tensor doesn't always divide evenly into the length of the output tensor. Refer to the code snippet below:
```python
T = 2**10
x = torch.zeros((1, 1, T), dtype=torch.float32)
x[0, 0, T//2] = 1
x_phis = []
x_psis = []
tfm = DTCWTForward(J=7, alternate_gh=True, include_scale=False)
x_phis, x_psis = tfm.forward(x)

print([x_psi.resolve_conj().numpy().shape[2] for x_psi in x_psis])
``` 
```
[1024, 1026, 515, 259, 131, 67, 35]
``` 
Solution :
Adjusted the padding parameter, still using zero-padding, but modified the amount of padding applied to the input to ensure the output length is "correct."
